app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :user, foreign_key: 'user_id'
```

Comment:
- [ ] Kindly note that the likes table will fail to locate the user's table because the foreign key is `author_id`, you need to indicate that the foreign key is `author_id`, like so: `belongs_to :author, foreign_key: 'author_id', class_name: "User"`